lib/streamlit/ScriptRunner.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Cool, I didn't know we could start removing these!

https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3

Reply:
Yeah, not necessary.  :)